detectron2/modeling/meta_arch/retinanet.py

```diff
@@ -92,12 +93,53 @@ def __init__(self, cfg):
             cfg.MODEL.RETINANET.IOU_LABELS,
             allow_low_quality_matches=True,
         )
+        self.vis_period = cfg.VIS_PERIOD
+        self.input_format = cfg.INPUT.FORMAT
```

Comment:
To have a consistent style, move them above in the `self.xxx = cfg.xxx` block and align the "=" sign.

Reply:
Fixed in 45d9600